cfme/infrastructure/virtual_machines.py

```diff
@@ -34,9 +34,9 @@
 from utils import version, deferred_verpick
 from widgetastic.widget import Text, View
```

Comment:
Can you move these up so the imports are correctly organized into the 3 groups (no comments, just import groups)

```python
# Import built-ins

# Import installed modules

# Import local code